.storybook/preview-head.html

```diff
@@ -4,7 +4,7 @@
 	- Whenever the `zeusLoader()` function is changed, we need to update this as well.
 -->
 <script
-  src="https://npm-test.zeustechnology.io/main.js"
+  src="https://test.zeustechnology.io/main.js"
```

Comment:
We want this to remain `npm-test` so that the published code keeps on the `npm-test` setup. It's better for making things stable.